components/radio/metadata/radio.yml

```
@@ -21,7 +21,7 @@ sections:
       - Read-only radio buttons do not have a focus ring, but the button should be focusable.]
 
       ### Invalid/Error State
-      - Invalid radio buttons are signified by including [HelpText](helptext.html) in a [FieldGroup](fieldgroup.html).
+      - Invalid radio buttons are signified by including [Help text](helptext.html) in a [Field group](fieldgroup.html). The `.is-invalid` class has been removed. See Field group for an example with an invalid radio group. 
```

Super minor and maybe not worth changing, but I'm curious if the bit about removing `.is-invalid` class is needed in this documentation. Seems like a release note comment not documentation?
```suggestion
      - Invalid radio buttons are signified by including [Help text](helptext.html) in a [Field group](fieldgroup.html). See Field group for an example with an invalid radio group. 
```

---

I'll defer to @pfulton on that one, but his part is under the "Migration Guide" which often lists things that have been changed or removed within other components.

This is a good point, and I have been wondering if there is a better way to present the notes under Migration Guide, across all components, since they keep getting longer and it is starting to become unclear as to what version what notes apply to---some migration notes may apply to a much older version that would be irrelevant to someone upgrading, but a more recent change may be mixed in the notes. I think more of a release notes style with versions that shows changes and any migration notes for that version might be an improvement.